### PR TITLE
Fixes #1652: Add AppLanguageFragment to the AppLanguageActivity

### LINKED ISSUE #1652: Add AppLanguageFragment to the AppLanguageActivity
The project follows the approach of using a fragment for each activity but that approach isn't used in `AppLanguageActivity`  
Add a new fragment to the `AppLanguageActivity`

### PR DESCRIPTION
**Pull request by MohamedMedhat1998**
## Explanation

- Fixes #1652 
- Added **AppLanguageFragment** to **AppLanguageActivity**
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by rt4914 — CHANGES_REQUESTED** [review]
Please have a look at https://github.com/oppia/oppia-android/pull/1577 review and update your PRs accordingly.

**Comment by MohamedMedhat1998** [comment]
> Please have a look at #1577 review and update your PRs accordingly.

Done, PTAL

File: app/src/test/java/org/oppia/app/testing/options/AppLanguageFragmentTest.kt [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import org.oppia.app.recyclerview.RecyclerViewMatcher.Companion.atPositionOnView
+import org.oppia.app.utility.OrientationChangeAction.Companion.orientationLandscape
+import org.robolectric.annotation.LooperMode
+
+@RunWith(AndroidJUnit4::class)
+class AppLanguageFragmentTest {
+
+  @Test
+  @LooperMode(LooperMode.Mode.PAUSED)
+  fun changeAppLanguageToFrench_changeConfiguration_selectedLanguageIsFrench() {
```
Comment (rt4914) [inline app/src/test/java/org/oppia/app/testing/options/AppLanguageFragmentTest.kt]: Please what you are testing first

`testAppLanguage_changeAppLanguageToFrench_changeConfiguration_selectedLanguageIsFrench `

File: app/src/test/java/org/oppia/app/testing/options/AppLanguageFragmentTest.kt [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          1, R.id.language_radio_button
+        )
+      ).perform(
+        click()
+      )
+      onView(isRoot()).perform(orientationLandscape())
+      onView(
+        atPositionOnView(
+          R.id.language_recycler_view,
+          1, R.id.language_radio_button
```
Comment (rt4914) [inline app/src/test/java/org/oppia/app/testing/options/AppLanguageFragmentTest.kt]: Nit:
```
1,
R.id.language_radio_button
```

File: app/src/test/java/org/oppia/app/testing/options/OptionsFragmentTest.kt [hunk 3]
```diff
@@ -57,6 +59,28 @@ class OptionsFragmentTest {
     }
   }
 
+  @Test
+  fun clickAppLanguage_checkSendingTheCorrectIntent() {
```
Comment (rt4914) [inline app/src/test/java/org/oppia/app/testing/options/OptionsFragmentTest.kt]: Ditto change for naming.

Also fix this for you StoryTextSize PR too.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Suggested nit change.

File: app/src/test/java/org/oppia/app/testing/options/OptionsFragmentTest.kt (hunk 3 above)
Comment (MohamedMedhat1998) [reply app/src/test/java/org/oppia/app/testing/options/OptionsFragmentTest.kt]: >Also fix this for you StoryTextSize PR too.

Done in #1579

File: app/src/test/java/org/oppia/app/testing/options/AppLanguageFragmentTest.kt (hunk 1 above)
Comment (MohamedMedhat1998) [reply app/src/test/java/org/oppia/app/testing/options/AppLanguageFragmentTest.kt]: Done

File: app/src/test/java/org/oppia/app/testing/options/AppLanguageFragmentTest.kt (hunk 2 above)
Comment (MohamedMedhat1998) [reply app/src/test/java/org/oppia/app/testing/options/AppLanguageFragmentTest.kt]: Done

File: app/src/test/java/org/oppia/app/testing/options/OptionsFragmentTest.kt (hunk 3 above)
Comment (MohamedMedhat1998) [reply app/src/test/java/org/oppia/app/testing/options/OptionsFragmentTest.kt]: >Ditto change for naming.

Done

**Review by rt4914 — APPROVED** [review]
LGTM

**Pull request merged**